packages/did/src/DidLinks/AccountLinks.chain.ts

```diff
@@ -62,7 +62,7 @@ export type EthereumAddress = HexString
 export type Address = KiltAddress | SubstrateAddress | EthereumAddress
 
 /// Type of signatures to link accounts to DIDs.
-export type SignatureType = MultiSignature['type'] | 'Ethereum'
+type SignatureType = Lowercase<MultiSignature['type']> | 'ethereum'
```

Comment:
we could explore if can't just use `KeyPair['type']` here

Reply:
Good idea!

Reply:
Seems to work in https://github.com/KILTprotocol/sdk-js/pull/588/commits/99fc11618c6ba81bab8437963d2cc939d63cf639